deploy/deploy-wmagent.sh

```diff
@@ -269,14 +271,20 @@ cd $BASE_DIR/deployment-$DEPLOY_TAG
 set -e
 for step in prep sw post; do
   echo -e "\n*** Deploying WMAgent: running $step step ***"
-  ./Deploy -R wmagent@$WMA_TAG -s $step -A $WMA_ARCH -r $REPO -t v$WMA_TAG $DEPLOY_DIR wmagent
+  ./Deploy -R ${RPM_NAME}@$WMA_TAG -s $step -A $WMA_ARCH -r $REPO -t v$WMA_TAG $DEPLOY_DIR ${RPM_NAME}
 done
 set +e
 
+echo -e "\n*** Creating wmagent symlink ***"
+cd $CURRENT_DIR
+ln -s ../sw/${WMA_ARCH}/cms/${RPM_NAME}/${WMA_TAG} apps/wmagent
```

Comment:
Here you seem to create the link name always as `apps/wmagent`, which to me seems logical, but down bellow you refer to it as `apps/${RPM_NAME}` (line 296 in the current commit)

Reply:
I had to create this symlink because some of those bin/* scripts look for files under `apps/wmagent`, which does not exist when we deploy `wmagentpy3`. 
The Deploy script automatically creates a symlink according to the package name. So, for python2, it won't make any difference since the symlink will be already there. For python3, we will have both symlinks pointing to the same directory structure.

In short, I could simply use `apps/wmagent` below as well, but I tried to keep it as consistent as possible.